gatsby-browser.js

```diff
@@ -1,4 +1,3 @@
-import './src/components/layout/fonts/fonts.css';
```

Comment:
With this removed, where do the fonts come from now? Is the `gatsby-ssr` script run for all requests?

Reply:
`gatsby-ssr` runs for all HTML files and adds the link to the font file into the header. The output is nearly the same as importing it here, but it's a bit more separated which makes it simpler to use in Storybook.